### DISCUSSION TITLE
Venus Master Fellowship Program

### DISCUSSION THREAD

**Discussion by Joss-Hua** [filecoin-project/community#363] [News and Events]
# Venus Master Fellowship Program - Incubation Center sub-project

We are glad to introduce the **Venus Master Fellowship Program** to you as part of our [Filecoin Storage Provider Incubation Center Program](https://venus.filecoin.io/incubation/). We have received positive feedback from the community. In case you haven't heard, check out the program [**here**](https://venus.filecoin.io/master/).

Venus Master Fellowship Program is a long-term community project and has made much progress as planned.

## This is important

Filecoin's mission is to create a decentralized, efficient, and robust foundation for humanity’s information. Based on this vision, different implementations of Filecoin is making progress.

[Venus](https://github.com/filecoin-project/venus), one of 4 implementations of Filecoin, is moving towards its goal through its distributed architecture. A large number of developers and storage providers would like to participate/contribute, but they can't start and progress effectively due to the lack of necessary tech support.

In the long run, Venus Masters is crucial to the success of Venus. It helps more storage providers smoothly join the Incubation Center or use Venus to run their Filecoin nodes, and helps more developers understand, familiarize and master Venus to jointly achieve this goal.

## How to work

For an introduction to the Venus Master Fellowship Program and an overview of its latest phase, please [check here](https://venus.filecoin.io/master/).

## Virtual organization

As pointed out in the project introduction, each stage will be limited to the selection of a fixed number of community members, but regardless of whether there is this label or not, Venus community members are developing towards learning together and helping each other.

## Venus Master Rewards

- Office hours with Venus team/core devs
- [Additional incentives](https://venus.filecoin.io/incubation/Rules.html#about-the-masters-fellowship-program)
- Constructive suggestions and latest progress

## How to apply

- Application has been closed
- Candidates will be notified via slack DM/Email
- Stay tuned for future cohort opportunities!

Comments are welcome in this thread.

**Comment by Joss-Hua** [comment]
## :handshake: Thanks

The members of phase 1 of Venus masters have been selected, and we will share their profiles with the community on this thread. We are excited to have them support Filecoin storage providers, clients, and developers.

**Comment by Joss-Hua** [comment]
## Meet the Venus Masters

Find them on GitHub, Filecoin Slack and etc if you have any questions!

**Comment by Joss-Hua** [comment]
# mariswang

Entrepreneurial developers

![mariswang](https://user-images.githubusercontent.com/75723122/139780783-634e6839-cc28-468c-9625-bd7e35f26825.png)

At present, it mainly focuses on the combination of big data and blockchain, and its interests include mining and playing ball.

**Location:** Beijing, China
**Slack:** mariswang
**Github:** [@maris205](https://github.com/maris205)
**Twitter:** [@maris205](https://twitter.com/maris205)
**LinkedIn:** https://www.linkedin.com/in/liang-wang-85076b47/

**Comment by Joss-Hua** [comment]
# cryptostack

![cryptostack](https://user-images.githubusercontent.com/75723122/139782444-d4c294c1-d7e9-487e-b7ce-e58bf0ebeac3.jpg)

Engaged in the study and development of blockchain-related fields, currently focusing on filecoin. I like music and running, but I prefer coding.

**Location:** Beijing, China
**Slack:** cryptostack-zou
**Github:** [@llifezou](https://github.com/llifezou)

**Comment by Joss-Hua** [comment]
# Jello Huang

Linux kernel and device driver developer

<img width="300" alt="TEHTVS1L6-U02BCHS0Q7R-213db72e2bdb-512" src="https://user-images.githubusercontent.com/75723122/139782775-9df5ac9f-eda5-4743-9a26-cb5b48cf97cb.png">

**Location:** Fuzhou, china
**Slack:** Jello Huang

**Reply by JelloHuang** [reply]
It is me,if u have any problem with filecoin & venus ,pls contact with me

**Reply by JelloHuang** [reply]
ZZ

**Comment by Joss-Hua** [comment]
# Dennis Zou

An old developer who has been in the Internet industry for more than ten years.

![TEHTVS1L6-U01U2M1GZL7-g8cdf2c64dfe-512副本](https://user-images.githubusercontent.com/75723122/139783175-59d5685b-da7d-426e-9f42-1153b11fbe61.jpg)

Wrote code;
Managed the server;
Led the technical team;
Developed large Internet applications;
Also managed large-scale clusters;
Like to study all kinds of new technologies.

**Location:** Shanghai, china
**Slack:** Dennis Zou
**Github:** [@denniszou](https://github.com/denniszou)
